lock-impl/src/main/java/com/palantir/lock/impl/LockServiceImpl.java

```diff
@@ -1241,10 +1243,28 @@ private String safeLockStats() {
     @Override
     public void close() {
         if (isShutDown.compareAndSet(false, true)) {
-            lockReapRunner.close();
-            threadInfoSnapshotManager.close();
-            blockingThreads.forEach(Thread::interrupt);
-            callOnClose.run();
+            List<Pair<String, Runnable>> closures = List.of(
+                    Pair.create("lockReapRunner", lockReapRunner::close),
+                    Pair.create("threadInfoSnapshotManager", threadInfoSnapshotManager::close),
+                    Pair.create("blockingThreads", () -> blockingThreads.forEach(Thread::interrupt)),
+                    Pair.create("callOnClose", callOnClose));
```

Comment:
I think you reference these as named actions later on, but it's probably worth changing the names to reflect the actions as opposed to the objects that were conceptually being closed (in particular for `blockingThreads`, it's unclear if you're calling `Thread.stop()` or interrupting or something else).